minimatch.js

```diff
@@ -708,11 +698,6 @@ function match (f, partial) {
 
   var options = this.options
 
-  // windows: need to use /, not \
-  if (path.sep !== '/') {
```

Comment:
I disagree. The idea here is to have a cross-platform API, with forward slash as path separator and backward slash as escape. That way you can still match literal forward (on Windows) and backward slashes by escaping them in the pattern.